.circleci/config.yml

```diff
@@ -1772,18 +1795,27 @@ workflows:
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
           requires: ['op-stack-go-docker-build']
           save_image_tag: <<pipeline.git.revision>> # for devnet later
+      - docker-build:
+          name: da-server-docker-build
+          docker_name: da-server
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          requires: ['op-stack-go-docker-build']
+          save_image_tag: <<pipeline.git.revision>> # for devnet later
       - cannon-prestate:
           requires:
             - go-mod-download
       - devnet:
+          name: devnet<< matrix.fpac >>
           matrix:
             parameters:
-              fpac: ["legacy", "fault-proofs"]
+              # TODO(10142): We need to re-enable "-fault-proofs" in the future
+              fpac: ["-legacy", "-plasma"]
```

Comment:
Personally, I'd omit the `-` prefix on all of these. Then you don't need to override the `name` as CircleCI will concat `{name}-{params}` for you (which is how we get `devnet-fault-proofs` currently.  Just update the other side to not have the `-` prefix either. Doesn't really matter either way though.

Reply:
Yea, will have to update the other side. We also need to update loading in the files, but that is easy